.travis.yml

```diff
@@ -10,6 +10,7 @@ before_install:
 install:
   - git clone https://github.com/canonical-webteam/documentation-builder
```

Comment:
How strange that we don't simply do `pip3 install -r requirements.txt` here. Do you know why that might not work?

Reply:
No idea. Let me try to update it to see